HotelPriceScout/Data/Interface/Dashboard.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Dapper;
+using DataAccessLibrary;
+using System;
+using System.Linq;
+using System.Collections;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+namespace HotelPriceScout.Data.Interface
+{
+    public class Dashboard
+    {
+        public string monthName { get; private set; } = "";
```

Comment:
Har i tænkt over, om der skal være nogle former for validation på nogle af alle de her properties? Det er ikke sikkert der skal på alle sammen, men det er muligt at der skal på nogle.

Reply:
Properties skal starte med stort forbogstav.